Services/UI/OJS.Services.Ui.Models/Contests/ContestServiceModel.cs

```diff
@@ -1,212 +1,210 @@
-﻿namespace OJS.Services.Ui.Models.Contests
+namespace OJS.Services.Ui.Models.Contests;
```

Comment:
Make changes like this in a different PR. As far as I can see, there are no changes to this file.

I can see the reason for making the changes, as they are needed for our code style, but they should be made clean.

Reply:
Got it.